public/js/profile.js

```diff
@@ -15,4 +14,5 @@ $.ajax({
       console.log(err);
       alert(err.responseText);
     });
+    location.reload();
```

Comment:
hey, I looked at this and I think it should be at line 11 otherwise it might reload before the ajax request is made.
I'm opening a pull request now that fixes the whole problem of adding more than one connections to the fav.